src/many/src/main.rs

```diff
@@ -160,7 +160,7 @@ struct ServerOpt {
     addr: SocketAddr,
 
     /// The name to give the server.
-    #[clap(long, short, default_value = "many-server")]
+    #[clap(long, short, default_value = "many-server-server")]
```

Comment:
That's a lot of servers.

Reply:
Refactoring in IntelliJ isn't _perfect_.